DataFormats/L1TCorrelator/src/TkBsCandidate.cc

```diff
@@ -0,0 +1,30 @@
+// -*- C++ -*-
+//
+// Package:     DataFormats/L1TrackTrigger
+// Class  :     TkBsCandidate
+//
+
+#include "DataFormats/L1TCorrelator/interface/TkPhiCandidate.h"
+#include "DataFormats/L1TCorrelator/interface/TkBsCandidate.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+using namespace l1t;
+
+TkBsCandidate::TkBsCandidate() {}
+TkBsCandidate::TkBsCandidate(const LorentzVector& p4, TkPhiCandidate cand1, TkPhiCandidate cand2) : L1Candidate(p4) {
+  phiCandList_.push_back(cand1);
+  phiCandList_.push_back(cand2);
+}
```

Comment:
This could be expressed also as
```suggestion
TkBsCandidate::TkBsCandidate(const LorentzVector& p4, TkPhiCandidate cand1, TkPhiCandidate cand2) : L1Candidate(p4), phiCandList_{cand1, cand2} {}
```

Reply:
done.